### PR TITLE
Add GA4 analytics tracking to print links

### PR DESCRIPTION
**Pull request by andysellick**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What
Add GA4 tracking to 'print this page' links on GOV.UK.

Note that these changes are only active on integration, not live.

Pages affected:

- detailed guides ([example detailed guide](https://www.gov.uk/guidance/register-a-trust-as-a-trustee))
- guides ([example guide](https://www.gov.uk/change-address-driving-licence/print))
- HMRC manuals ([example manual](https://www.gov.uk/hmrc-internal-manuals/employment-income-manual) and [example manual section](https://www.gov.uk/hmrc-internal-manuals/employment-income-manual/eim00100) and [example updates page](https://www.gov.uk/hmrc-internal-manuals/trust-registration-service-manual/updates))
- HTML publication ([example HTML publication](https://www.gov.uk/government/publications/budget-2016-documents/budget-2016))
- manuals ([example manual section](https://www.gov.uk/guidance/the-highway-code/general-rules-techniques-and-advice-for-all-drivers-and-riders-103-to-158) and [example manual](https://www.gov.uk/guidance/academy-trust-handbook))
- travel advice ([example travel advice](https://www.gov.uk/foreign-travel-advice/usa/print))

In some of these commits I've removed the explicit passing of `module: "print-link"` in the data attributes. This is because it is not needed - [the component adds this itself](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_print_link.html.erb#L9) regardless of what is passed.

## Why
Part of the GA4 test rollout.

## Visual changes
None.

Trello card: https://trello.com/c/GlsJk20k/350-print-this-page

### CODE REVIEW

File: app/views/content_items/travel_advice.html+print.erb [hunk 1]
```diff
@@ -20,10 +20,17 @@
...
+        module: "ga4-event-tracker",
         "track-category": "printButton",
         "track-action": "clicked",
         "track-label": t("components.print_link.text"),
-        module: "print-link"
+        ga4: {
+          event_name: 'print_page',
+          type: 'Print this page',
+          index: 1,
+          index_total: 1,
```
Comment (JamesCGDS) [inline app/views/content_items/travel_advice.html+print.erb:31]: Would this need to be `index_total: 2`?

File: app/views/content_items/travel_advice.html+print.erb [hunk 2]
```diff
@@ -45,10 +52,17 @@
...
```
Comment (JamesCGDS) [inline app/views/content_items/travel_advice.html+print.erb:63]: Just to check: would this need to be `index: 2` and `index_total`: 2?

File: app/views/content_items/travel_advice.html+print.erb (hunk 1 above)
Comment (andysellick) [reply app/views/content_items/travel_advice.html+print.erb]: Yes, great spot, thanks, have updated.

File: app/views/content_items/travel_advice.html+print.erb (hunk 2 above)
Comment (andysellick) [reply app/views/content_items/travel_advice.html+print.erb]: Yes, great spot, thanks, have updated.

**Review by JamesCGDS — APPROVED** [review]

**Pull request merged**